praktikumsplaner-frontend/frontend/tests/unit/rules.spec.ts

```diff
@@ -49,3 +48,30 @@ describe("rules test", () => {
         expect(txtRule(undefined)).toBe(errorMessage);
     });
 });
+describe("rules maxLength test", () => {
+    it("tests maxLengthRule return true", () => {
+        const validationRules = useRules();
+
+        const txtRule = validationRules.maxLengthRule(10, errorMessage);
+        const txt10 = "Lorem ipsu";
+
+        expect(txtRule(txt10)).toBe(true);
+    });
+    it("tests maxLengthRule return error", () => {
+        const validationRules = useRules();
+
+        const txtRule = validationRules.maxLengthRule(10, errorMessage);
+        const txt11 = "Lorem ipsum";
+
+        expect(txtRule(txt11)).toBe(errorMessage);
+    });
+    it("tests maxLengthRule return error", () => {
+        const validationRules = useRules();
+
+        const txtRule = validationRules.maxLengthRule(10, errorMessage);
+
+        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+        // @ts-ignore
```

Comment:
with that ignore comments the problems got hidden not solved. Please fix the problem.